packages/lesswrong/components/posts/PostsPage/PostBodyPrefix.tsx

```diff
@@ -15,26 +16,43 @@ const styles = (theme: ThemeType): JssStyles => ({
     ...theme.typography.contentNotice,
     ...theme.typography.postStyle
   },
+  infoIcon: {
+    width: 16,
+    height: 16,
+    verticalAlign: "top",
+    color: "rgba(0,0,0,.4)",
+  },
 });
 
 const PostBodyPrefix = ({post, query, classes}: {
   post: PostsWithNavigation|PostsWithNavigationAndRevision,
   query?: any,
   classes: ClassesType,
 }) => {
-  const { AlignmentCrosspostMessage, LinkPostMessage, PostsRevisionMessage, HoverPreviewLink, ReviewPostButton} = Components;
+  const { AlignmentCrosspostMessage, LinkPostMessage, PostsRevisionMessage, LWTooltip} = Components;
   
   return <>
     {/* disabled except during Review */}
-    {(post.nominationCount2019 >= 2) && <div className={classes.reviewInfo}>
+    {/* {(post.nominationCount2019 >= 2) && <div className={classes.reviewInfo}>
       <div className={classes.reviewLabel}>
         This post has been nominated for the <HoverPreviewLink href={'/posts/QFBEjjAvT6KbaA3dY/the-lesswrong-2019-review'} id="QFBEjjAvT6KbaA3dY" innerHTML={"2019 Review"}/>
       </div>
       <ReviewPostButton post={post} reviewMessage="Write a Review" year="2019"/>
-    </div>}
+    </div>} */}
 
     <AlignmentCrosspostMessage post={post} />
-    { post.authorIsUnreviewed && !post.draft && <div className={classes.contentNotice}>This post is awaiting moderator approval</div>}
+    {/* ea-forum-look-here */}
```

Comment:
Also need to verify this makes sense

Reply:
I think this will be good, just need to replace LW with {siteTitleSetting.get()}